QUARKUS-2394.md

```diff
@@ -0,0 +1,39 @@
+# QUARKUS-2394 Drop OpenTelemetry Jaeger exporter
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2394
+
+Opentelemetry Jaeger exported has been deprecated from an official Quarkus support point of view. The extension `quarkus-opentelemetry-exporter-jaeger` will be moved to Quarkiverse and `quarkus-opentelemetry-exporter-otlp` extension has been promoted as the supported one. 
```

Comment:
```suggestion
OpenTelemetry Jaeger exporter has tech-preview support in RHBQ 2.7. Starting with RHBQ 2.13 the tech-preview support is dropped. The extension `quarkus-opentelemetry-exporter-jaeger` is planned to be moved to Quarkiverse in the future, `quarkus-opentelemetry-exporter-otlp` extension is the one to be used instead of Jaeger exporter. 
```